src/inference/dev_api/openvino/runtime/system_conf.hpp

```diff
@@ -228,8 +228,9 @@ enum ColumnOfProcessorTypeTable {
  * @brief      Definition of CPU_MAP_USED_FLAG column in CPU mapping table.
  */
 enum ProcessorUseStatus {
-    NOT_USED = -1,  //!< Processor is not bound to thread
-    CPU_USED = 1,   //!< CPU is in using
+    CPU_BLOCKED = -100,  //!< Processor is blocked to use
+    NOT_USED = -1,       //!< Processor is not bound to thread
+    CPU_USED = 1,        //!< CPU is in using
```

Comment:
Why not use 0, 1, 2  as enum value, maybe it can benefit a little memory optimization than -100. -1, 1

Reply:
Positive number of CPU_USED will be used to identify using by different device or different model in the future.